cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/PartialPostProcessingInput.view.yaml

```diff
@@ -1,54 +1,54 @@
-space: '{{powerops_models}}'
+space: "{{powerops_models}}"
```

Comment:
What about changing the name `PartialPostProcessingInput` to `PartialBidMatrixPostProcessingInput`? Same with the Output object?